jsk_fetch_robot/jsk_fetch_diagnosis/package.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>jsk_fetch_diagnosis</name>
+  <version>0.0.0</version>
+  <description>The fetch_diagnosis package</description>
+
+  <maintainer email="shinjo@todo.todo">shinjo</maintainer>
```

Comment:
Use real name and address `<maintainer email=“sktometometo@gmail.com or jsk address”>Koki Shinjo</maintainer>`

Reply:
set maintainer information